core/src/serve_repair.rs

```diff
@@ -323,31 +327,35 @@ impl ServeRepair {
         requests_receiver: &PacketBatchReceiver,
         response_sender: &PacketBatchSender,
         stats: &mut ServeRepairStats,
-        packet_threshold: &mut DynamicPacketToProcessThreshold,
```

Comment:
`ancestor_hashes_service` using the same struct.
Doesn't that have the same issue?

Reply:
Yeah, seems like it could be another PR though for that service